python/cog/server/helpers.py

```diff
@@ -67,6 +60,13 @@ def original(self) -> TextIO:
             raise RuntimeError("you must call wrap() before using original")
         return self._original_fp
 
+    def _get_max_pipe_size(self) -> int:
```

Comment:
this doesn't use self though?

Reply:
I made it an instance method, so self is a required parameter. Could alternatively be a class method or underscored method at top-level.